app/Transformers/GroupTransformer.php

```diff
@@ -23,4 +24,12 @@ public function transform(Group $group)
             'short_name' => $group->short_name,
         ];
     }
+
+    public function includeDescription(Group $group)
+    {
+        return $this->primitive([
```

Comment:
the typing is simpler if it returns null instead of array with null values `$this->primitive($group->group_desc === null ? null : [...])`